src/System.Data.SqlClient/tests/ManualTests/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
+      netstandard;
```

Comment:
Can you run `msbuild /t:UpdateVSConfigurations build.proj` from the root and check the csproj changes in as well?